http-utils/src/main/java/com/facebook/airlift/http/utils/jetty/ConcurrentScheduler.java

```diff
@@ -0,0 +1,68 @@
+package com.facebook.airlift.http.utils.jetty;
+
+import com.facebook.airlift.concurrent.ConcurrentScheduledExecutor;
+import org.eclipse.jetty.util.component.AbstractLifeCycle;
+import org.eclipse.jetty.util.thread.Scheduler;
+
+import java.util.Optional;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.Math.max;
+import static java.util.Objects.requireNonNull;
+
+public class ConcurrentScheduler
+        extends AbstractLifeCycle
+        implements Scheduler
+{
+    private final int schedulerCount;
+    private final int threadsPerScheduler;
+    private final String threadBaseName;
+
+    private Optional<ConcurrentScheduledExecutor> concurrentScheduler;
+
+    public ConcurrentScheduler(int schedulerCount, int threadsPerScheduler, String threadBaseName)
+    {
+        checkArgument(schedulerCount > 0, "schedulerCount must be at least one");
+        checkArgument(threadsPerScheduler > 0, "threadsPerScheduler must be at least one");
+
+        this.schedulerCount = schedulerCount;
+        this.threadsPerScheduler = threadsPerScheduler;
+        this.threadBaseName = requireNonNull(threadBaseName, "threadBaseName is null");
+        concurrentScheduler = Optional.empty();
+    }
+
+    public static ConcurrentScheduler createConcurrentScheduler(String threadBaseName, int concurrency, int totalThreads)
+    {
+        checkArgument(concurrency >= 1, "concurrency must be at least one");
+        int threadsPerScheduler = max(1, totalThreads / concurrency);
+        return new ConcurrentScheduler(concurrency, threadsPerScheduler, threadBaseName);
+    }
+
+    @Override
+    protected void doStart()
+    {
+        concurrentScheduler = Optional.of(new ConcurrentScheduledExecutor(
```

Comment:
nit: don't wrap

We are trying to avoid this style

```
callSomething(arg1, arg2) - OK
callSomething(
  arg1, 
  arg2) - OK
callSomething(
  arg1, arg2) - NO
callSomething(arg1, 
  arg2) - NO
```